tests/_data/fixtures/Migrations/OrdersProductsMigration.php

```diff
@@ -71,18 +71,18 @@ protected function getSqlPgsql(): array
     {
         return [
             "
-drop table if exists private.co_orders_x_products;
+drop table if exists public.co_orders_x_products;
```

Comment:
@Jeckerson This table needs to remain in the "private" schema. Because I ran a test in the JoinCest file which checks the function ManyToMany with different schemas #14854. Before this test and bug fix, Phalcon didn't detect both schemas.
So can you put back the private schema only for this table. 
Thank you

Reply:
@Jeckerson We can also merge this and let @jenovateurs change this in combination with the other tests.

Reply:
@jenovateurs The problem is that in PostgreSQL we do not have private schema. I'll see if possible to adapt and work in both: MySQL and PGSQL.

Reply:
@Jeckerson You need to create it, I have already done this in my local branch. Like I said @ruudboon I can change this by doing the other issue.

Reply:
@jenovateurs alright, then make sense to you create an PR that will be merged, and then I'll merge it with current PR.

Reply:
@Jeckerson If we merge the two PR, it should be work. Do you want me to update the GitHub Action to run the PostgreSQL test in my PR?